packages/renderer/src/lib/container/ContainerExport.svelte

```diff
@@ -0,0 +1,120 @@
+<script lang="ts">
+import { faDownload, faFolderOpen } from '@fortawesome/free-solid-svg-icons';
+import { Input } from '@podman-desktop/ui-svelte';
+import { onMount } from 'svelte';
+import { router } from 'tinro';
+
+import { exportContainerInfo } from '/@/stores/export-container-store';
+import { createTask } from '/@/stores/tasks';
+
+import Button from '../ui/Button.svelte';
+import ErrorMessage from '../ui/ErrorMessage.svelte';
+import FormPage from '../ui/FormPage.svelte';
+import { Uri } from '../uri/Uri';
+import type { ContainerInfoUI } from './ContainerInfoUI';
+
+let container: ContainerInfoUI;
+
+let invalidName = false;
+let invalidFolder = true;
+let outputTarget = '';
+let outputUri: Uri;
+let exportedError = '';
+$: invalidFields = invalidName || invalidFolder;
+
+onMount(async () => {
+  // grab current value
+  container = $exportContainerInfo;
+  if (!container) {
+    // go back to containers list
+    router.goto('/containers/');
+  }
+});
+
+async function selectFolderPath() {
+  const result = await window.saveDialog({
+    title: 'Select the directory where to export the container content',
```

Comment:
By default the output folder is 'Unititled'. Could we change this to the name of the image (or at least be unique)?